inc/drivers/MicroBitPin.h

```diff
@@ -95,8 +95,8 @@ class MicroBitPin : public MicroBitComponent
 {
     // The mbed object looking after this pin at any point in time (untyped due to dynamic behaviour).
     void *pin;
-
     PinCapability capability;
+    uint8_t pullMode;
```

Comment:
checked that - fits within current word alignment bounds, so adds no additional overhead.